src/status_im/contexts/wallet/add_address_to_watch/confirm_address/view.cljs

```diff
@@ -16,9 +16,9 @@
   []
   (let [{:keys [address]}  (rf/sub [:get-screen-params])
         number-of-accounts (count (rf/sub [:wallet/watch-only-accounts]))
-        account-name       (reagent/atom "")
         placeholder        (i18n/label :t/default-watched-address-placeholder
                                        {:number (inc number-of-accounts)})
+        account-name       (reagent/atom placeholder)
```

Comment:
we don't want this @ulisesmac - the designs/ approach has been updated that we just use a placeholder instead now and no default name 👍 
In fact the placeholder can simply be "Account Name" and "Watched Address Name"

Reply:
Thanks @J-Son89 

Do we have the link to those designs?

I've done this because of:
- #18856
It contains a link to designs, probably they are also outdated?

Reply:
Yeah it's out of date with respect to that detail 👌

Reply:
ok, I'll fix according to what you said :+1:

Reply:
done